ci/tests_all_in_schedule.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -euo pipefail
+
+# shellcheck disable=SC1091
+source ci/ci_helpers.sh
+cd src/test/regress
+
+for test_dir in sql spec; do
```

Comment:
what about `input` directory?

Reply:
I guess `input` directory is becoming obsolete, so I wouldn't mind having it if it  is hard to add

Reply:
In the past, it was the only way to run COPY on distributed tables, nowadays we can use `generate_series` to generate data, so not many of us use it anymore. 

But, I realized that we added `hyperscale_tutorial.source` to there as the tutorial has some COPY commands.

Still, it seems OK not to have `input` directly if it is going to complicate things, as we add new tests to `input` rarely.